api/util/util_test.go

```diff
@@ -65,4 +39,134 @@ var _ = Describe("Util", func() {
 		})
 	})
 
+	Describe("HandlePrivateSSHKey", func() {
+
+		rawString := "echo 'GIT_PRIVATE_SSH_KEY' > ~/.ssh/huskyci_id_rsa &&"
+		expectedNotEmpty := "echo 'PRIVKEYTEST' > ~/.ssh/huskyci_id_rsa &&"
+		expectedEmpty := "echo '' > ~/.ssh/huskyci_id_rsa &&"
+
+		Context("When rawString and HUSKYCI_API_GIT_PRIVATE_SSH_KEY are not empty", func() {
+			It("Should return a string based on these params", func() {
+				os.Setenv("HUSKYCI_API_GIT_PRIVATE_SSH_KEY", "PRIVKEYTEST")
+				Expect(util.HandlePrivateSSHKey(rawString)).To(Equal(expectedNotEmpty))
+			})
+		})
+		Context("When rawString is empty and HUSKYCI_API_GIT_PRIVATE_SSH_KEY is not empty", func() {
+			It("Should return an empty string.", func() {
+				Expect(util.HandlePrivateSSHKey("")).To(Equal(""))
+			})
+		})
+		Context("When rawString is not empty and HUSKYCI_API_GIT_PRIVATE_SSH_KEY is empty", func() {
+			It("Should return a string based on these params.", func() {
+				os.Unsetenv("HUSKYCI_API_GIT_PRIVATE_SSH_KEY")
+				Expect(util.HandlePrivateSSHKey(rawString)).To(Equal(expectedEmpty))
+			})
+		})
+		Context("When rawString and HUSKYCI_API_GIT_PRIVATE_SSH_KEY are empty", func() {
+			It("Should return an empty string.", func() {
+				Expect(util.HandlePrivateSSHKey("")).To(Equal(""))
+			})
+		})
+	})
+
+	Describe("GetLastLine", func() {
+
+		rawString := `Warning: unpinned requirement
+{"name":"enry", "vulnerability":"low"}`
+		expected := `{"name":"enry", "vulnerability":"low"}`
+
+		Context("When rawString is not empty", func() {
+			It("Should return the string that is in the last position", func() {
+				Expect(util.GetLastLine(rawString)).To(Equal(expected))
+			})
+		})
+		Context("When rawString is empty", func() {
+			It("Should return an empty string.", func() {
+				Expect(util.GetLastLine("")).To(Equal(""))
+			})
+		})
+	})
+
+	Describe("GetAllLinesButLast", func() {
+
+		rawString := `Line1
+Line2
+Line3
+Line4`
+		expected := []string{"Line1", "Line2", "Line3"}
+
+		Context("When rawString is not empty", func() {
+			It("Should return the slice of strings except the last line", func() {
+				Expect(util.GetAllLinesButLast(rawString)).To(Equal(expected))
+			})
+		})
+		Context("When rawString is empty", func() {
+			It("Should return an empty slice of string.", func() {
+				Expect(util.GetAllLinesButLast("")).To(Equal([]string{}))
+			})
+		})
+	})
+
+	Describe("RemoveDuplicates", func() {
+
+		rawSliceString := []string{"item1", "item2", "item3", "item1", "item2"}
+		expected := []string{"item1", "item2", "item3"}
+
+		Context("When rawSliceString is not empty", func() {
+			It("Should return the slice of strings except the last line", func() {
```

Comment:
Can you change "Should return the slice of strings except the last line" for "Should return slice of non-duplicate elements" or something like it?

Reply:
Absolutely! Just did it! Thanks for this 🙃